services/upload/src/test/java/app/coronawarn/server/services/federation/upload/utils/MockData.java

```diff
@@ -0,0 +1,74 @@
+/*-
+ * ---license-start
+ * Corona-Warn-App
+ * ---
+ * Copyright (C) 2020 SAP SE and all other contributors
+ * ---
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ---license-end
+ */
+
+package app.coronawarn.server.services.federation.upload.utils;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import app.coronawarn.server.common.persistence.domain.DiagnosisKey;
+import app.coronawarn.server.common.persistence.domain.FederationUploadKey;
+import app.coronawarn.server.common.protocols.external.exposurenotification.ReportType;
+
+public class MockData {
+
+  public static final String TEST_ORIGIN_COUNTRY = "DE";
+
+  public static List<FederationUploadKey> generateRandomUploadKeys(boolean consentToShare, int numberOfKeys) {
+    List<FederationUploadKey> fakeKeys = new ArrayList<>();
+    while (numberOfKeys > 0) {
+      fakeKeys.add(generateRandomUploadKey(consentToShare));
+      numberOfKeys--;
```

Comment:
Avoid mutating method arguments. Use `for`-loop or `IntStream.range(0, numberOfKeys)` instead.